app/assets/stylesheets/content/work_packages/inplace_editing/_edit_fields.sass

```diff
@@ -91,8 +91,13 @@
       padding-right: 0
 
 // Full width container
-td.subject .wp-table--cell-container .wp-edit-field:not(.wp-table--cell-span)
-  display: inline-block
+td.subject .wp-table--cell-container
+  max-width: inherit
+
+  .wp-edit-field:not(.wp-table--cell-span)
+    display: inline-block
+    width: 100%
```

Comment:
The `width: 100%` causes the edit field to be larger than actually possible (100% includes the width of the hierarchy indicator).


![bildschirmfoto 2017-06-01 um 15 28 34](https://cloud.githubusercontent.com/assets/459462/26681907/fea97998-46de-11e7-9325-bbdc9dc4160b.png)